src/cmds/scripts/pbs_preuninstall

```diff
@@ -61,9 +62,10 @@ server|execution)
         echo "have systemd"
         systemctl disable pbs
         rm -f /usr/lib/systemd/system-preset/95-pbs.preset
-        systemctl stop pbs_jobs.service
-        [ -d /sys/fs/cgroup/cpuset/pbs_jobs.service/jobid ] && rmdir /sys/fs/cgroup/cpuset/pbs_jobs.service/jobid
-        [ -d /sys/fs/cgroup/cpuset/pbs_jobs.service ] && rmdir /sys/fs/cgroup/cpuset/pbs_jobs.service
+        service_name=`cat ${pbs_home}/server_priv/hooks/pbs_cgroups.CF | python -c "import sys, json; print(json.load(sys.stdin)['cgroup_prefix'])"`
```

Comment:
Need to guard against error doing cat or python. Maybe check first for existence of ${pbs_home}/server_priv/hoks/pbs_cgroups.CF, and also if 'python' is in path.